matrixConsole/AppDelegate.m

```diff
@@ -362,6 +372,27 @@ - (void)applicationWillTerminate:(UIApplication *)application
     // Called when the application is about to terminate. Save data if appropriate. See also applicationDidEnterBackground:.
 }
 
+#pragma mark - Handling force touch shortcuts
+
+- (void)application:(UIApplication *)application performActionForShortcutItem:(UIApplicationShortcutItem *)shortcutItem completionHandler:(void (^)(BOOL))completionHandler {
+    
+    UIStoryboard *storyboard = [UIStoryboard storyboardWithName:@"Main" bundle:[NSBundle mainBundle]];
+    
+    UITabBarController *tabBarVC = [storyboard instantiateViewControllerWithIdentifier:@"MasterTabBarController"];
+    
+    if ([shortcutItem.type containsString:@"home"])
+        tabBarVC.selectedIndex = 0;
+    else if ([shortcutItem.type containsString:@"recents"])
+        tabBarVC.selectedIndex = 1;
+    else if ([shortcutItem.type containsString:@"contacts"])
+        tabBarVC.selectedIndex = 2;
+    else if ([shortcutItem.type containsString:@"settings"])
+        tabBarVC.selectedIndex = 3;
```

Comment:
You should use here the constant TABBAR_SETTINGS_INDEX defined in "MasterTabBarController.h".